__tests__/app.test.js

```diff
@@ -122,3 +124,23 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("status(200),responds with matching article with comment count added", () => {
+    return request(app)
+      .get(`/api/articles/1`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
```

Comment:
I can see you changed your original test to use a different matcher to make it less brittle but then you used toEqul here which would face the same issue should you add newer tests. You probably don't need to test for ALL the properties since we know these are here from your previous test, but no harm done :)